miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/view/EditProjectController.java

```diff
@@ -76,6 +76,15 @@ public ModelAndView setupForm(ModelMap model) throws IOException {
     return setupForm(new ProjectImpl(), model);
   }
 
+  @GetMapping("/shortname/{shortName}")
+  public ModelAndView byProjectShortName(@PathVariable String shortName, ModelMap model) throws IOException {
+    Project project = projectService.getProjectByCode(shortName);
+    if (project == null) {
+      throw new NotFoundException("No project found for short name " + shortName);
+    }
+    return new ModelAndView("forward:/project/code/%s".formatted(shortName), model);
```

Comment:
This method should only do the redirect. Validation will be handled after the redirect in `byProjectCode` below

Reply:
Also, it should be "redirect:", not "forward:", sorry!